framework/CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ Yii Framework 2 Change Log
 - Bug #14492: Fixed error handler not escaping error info debug mode (samdark)
 - Chg #14487: Changed i18n message error to warning (dmirogin)
 - Enh #7823: Added `yii\filters\AjaxFilter` filter (dmirogin)
+- Enh #14546: Added `dataFolder` property into `BaseActiveFixture` (leandrogehlen)
```

Comment:
Currently, the fixtures directory is `@tests/fixtures/data`. The default directory for it in codeception is `@tests/_data` with this implementation it's possitble to change Yii2 module to use `@tests/_data` as default directory